### ISSUE TITLE
Pre-release won't launch.

### ISSUE THREAD

**Issue by sidav** [AllureOfTheStars/Allure#114]
What I did:
1. Got the latest pre-release archive (Ubuntu); 
2. Unpacked it to `~/programs/`
3. ` cd ~/programs/AllureOfTheStars`
4. `./Allure`
Got the following error:
Allure: user error (Font file does not exist: /home/mikolaj/r/Allure/./GameDefinition/fonts/16x16xw.bdf)
Seems like the file paths are hard-coded. Is it intended?

**Comment by Mikolaj** [comment]
Oh dear. No, it's not intended. Thank you for the report --- will fix. I wonder why it works fine on Windows, despite being generated by the same makefile...

**Comment by Mikolaj** [comment]
Oh, I see, Windows uses a variant with v1-build and Ubuntu with v2-build, so there is a difference.

**Comment by Mikolaj** [comment]
@sidav: I've updated the package. Please let me know if it works.

I've temporarily fixed it, for this build, by hardwiring `rfontDir = "GameDefinition/fonts"`, but please let's leave the ticket open so that I can come up with a correct solution that works both with Debian packages and in-place builds and does so using `cabal v2-build`.

**Comment by sidav** [comment]
Yup, all is working now. Thank you!
As you requested, I'll leave this ticket open.

**Comment by Mikolaj** [comment]
OK, I've sorted it out for good. Changed the build scripts and fixed the broken travis and appveyor CIs.

Apparently some things changed on the CI servers, but the main issue was as reported in this ticket: finding the data files was too fragile so, advised by @phadej, I've simplified this by embedding all the font files in the executable (default config is embedded as well, so there's no more data files that would need to be looked up in some directory that may be different for the main game executable and different for the engine and it's not clear which one is chosen, etc.).

Thank you once more for the report. Closing.
